examples/templates/Example.js

```diff
@@ -206,6 +217,32 @@
             if ( optionalArgs )
                 osg.extend( optionsDebug, optionalArgs );
 
+            if ( !this._programRTT ) {
```

Comment:
it makes rendering of the debug quad RTT independent of the scene tree, so that it's drag and drop code, without needing user care.
(without that code I got black quads)